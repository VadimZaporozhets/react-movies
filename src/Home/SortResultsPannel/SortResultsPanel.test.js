import React from 'react';
import { SortResultsPanelComponent } from './index';
import { shallow } from 'enzyme';

describe('<SortResultsPanelComponent />', () => {
    it('should render sort panel component', () => {
        const props = {
            classes: {
                marginRight: {},
                paperWrapper: {},
                marginBottom: {}
            }
        };

        const wrapper = shallow(<SortResultsPanelComponent {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
