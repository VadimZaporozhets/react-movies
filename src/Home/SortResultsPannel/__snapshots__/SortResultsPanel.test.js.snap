// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SortResultsPanelComponent /> should render sort panel component 1`] = `
<WithStyles(Grid)
  className={Object {}}
  container={true}
  spacing={40}
>
  <WithStyles(Grid)
    item={true}
    xs={12}
  >
    <WithStyles(Paper)
      className={Object {}}
      elevation={1}
    >
      <WithStyles(Grid)
        container={true}
        spacing={40}
      >
        <WithStyles(Grid)
          alignItems="center"
          container={true}
          item={true}
          md={6}
          sm={12}
        >
          <WithStyles(Typography)
            className="margin-right"
            variant="button"
          >
            Found 20 movies
          </WithStyles(Typography)>
        </WithStyles(Grid)>
        <WithStyles(Grid)
          item={true}
          md={6}
          sm={12}
        >
          <WithStyles(Typography)
            className={Object {}}
            gutterBottom={true}
            inline={true}
            variant="button"
          >
            Sort by:
          </WithStyles(Typography)>
          <WithStyles(Button)
            className={Object {}}
            color="primary"
            size="small"
            variant="outlined"
          >
            Release date
          </WithStyles(Button)>
          <WithStyles(Button)
            color="primary"
            size="small"
            variant="outlined"
          >
            Rating
          </WithStyles(Button)>
        </WithStyles(Grid)>
      </WithStyles(Grid)>
    </WithStyles(Paper)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
